### DISCUSSION TITLE
Alternative "Hide Under Value" For Stackable Items in Grount Items Plugin

### DISCUSSION THREAD

**Discussion by Jaqzz** [runelite/runelite#16189] [Ideas]
Stackable items are, at least for me, frequently worth picking up at smaller values when compared to non stackable items, so setting my "Hide Under Value" to get rid of the visual clutter of non-stackable items I don't want to pick up frequently means either right clicking death piles to check like vanilla or putting the specific items I'm looking for on the highlited item list, which is getting kind of bloated. This is especially true for my ironman, who now gets a cheerful notification any time a lizardman fang or herb seed gets dropped because of how difficult it is to see those. It isn't terrible how it is now, but when I'm fighting something that drops stuff that's actually extremely useful it's annoying to have the same color/notification for a cheap but useful item I need to collect a lot of and the big ticket item that I've been specifically hunting.

So, actual suggestion:

- Add a "Stackable Hide Under Value" that when set to a nonzero number overrides the default behavior of the Hide Under Value for anything that stacks (seeds, runes, coins, etc)
- Add a toggle "Count _______ as stackable" for each of uncut gems, grimy herbs, and any other non-stackable object that can be stored in a bag.

**Comment by Enriath** [comment]
See #13611 & #14013
